src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -313,13 +445,26 @@ label {
 }
 
 // disable the clickable v-textfield label
-:deep(.identifier-textfield label) {
+:deep(.incorporation-number label) {
   pointer-events: none;
 }
 
-// raise the checkbox input to align with the label
-:deep(.v-input--checkbox .v-input--selection-controls__input) {
-  margin-top: -1rem;
+// style the radio buttons
```

Comment:
This is needed because Yui wanted the radio buttons to be extra wide, and I also wanted them to be responsive.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/4d19df5e-b139-46d0-8e44-da6ffbd0792f)

![image](https://github.com/bcgov/business-create-ui/assets/10002889/5dca954d-0da3-411a-b60d-5fa076af556f)